src/class-config.php

```diff
@@ -224,7 +224,7 @@ protected function get_acf_field_value( $root, $acf_field ) {
 					$id = 'comment_' . absint( $root->comment_ID );
 					break;
 				default:
-					$id = null;
+					$id = apply_filters( 'graphql_acf_get_root_id', null, $root );
```

Comment:
Need a DocBloc for the filter:

```suggestion
					$id = null;
					/**
					 * Filters the root ID, allowing additional Models the ability to provide a way to resolve their ID
					 *
					 * @param int $id The ID of the object. Default null
					 * @param mixed $root The Root object being resolved. The ID is typically a property of this object
					 * 
					 */
					$id = apply_filters( 'graphql_acf_get_root_id', $id, $root );
```

Also, I wonder if we actually apply this filter _after_ the Switch statement. There's a _chance_ that someone may still want to override the ID for other Models too.